source/dub/dub.d

```diff
@@ -1931,4 +1954,12 @@ private class DubConfig {
 		if (m_parentConfig) return m_parentConfig.defaultPostRunEnvironments;
 		return null;
 	}
+
+	@property string dubHome()
+	const {
+		if(auto pv = "dubHome" in m_data)
```

Comment:
should the setting for this be called `dubHome` or something else?

Reply:
Can we make things a bit more hierarchical, now that we have the ability to (easily) do so ?
Like `core.author` in `git`, you know.